README.md

```diff
@@ -176,8 +176,38 @@ path = ArtifactoryPath(
     "http://repo.jfrog.org/artifactory/distributions/org/apache/tomcat/apache-tomcat-7.0.11.tar.gz"
 )
 
-with open("tomcat.tar.gz", "wb") as out:
+# download by providing path to output file and use default chunk 1024
+path.writeto(output="tomcat.tar.gz")
+
+# download and suppress progress messages
+path.writeto(output="tomcat2.tar.gz", progress_func=None)
+
+# download by providing output as file object and specify chunk size
+with open("tomcat3.tar.gz", "wb") as out:
     path.writeto(out, chunk_size=256)
+
+# select another method for progress reporting (printing) eg print instead of logging
+from artifactory import print_download_progress
+
+path.writeto(
+    output="tomcat4.tar.gz",
+    progress_func=lambda x, y: print_download_progress(x, y, print_method=print),
```

Comment:
It look like I'd rather create a new one function then use this one with a lot of configuration parameters :)

Let's use `logging` by default and if users really need it - they'll copy\paste and create the `print_progress_func`

Reply:
@allburov 
I am working currently on 3 independent projects within organization and see that in all of them people need same functionality to print. But as you mentioned earlier all of them used different ways. 

What I want to say that we need to allow people an easy way of reporting status without redeclaring their own function.
In this case adding one argument seems like a reasonable compromise. I would ask to keep it

Reply:
What is the difference between logging and print in case of CLI application? Could you explain what is confusing you with logging?

Reply:
Now I see, we look from different perspectives. This project is also well used not only as CLI application but just as python module in larger eco systems, for example for CI CD.
In fact everywhere we see output of the download progress, users cannot just seat and wait without understanding where they are

Again, adding one additional argument for me looks like a good compromise

Reply:
Users can configure `logging` and it'll show messages in stdout\stderr. It's exaclty what `print` do.
https://stackoverflow.com/questions/14058453/making-python-loggers-output-all-messages-to-stdout-in-addition-to-log-file

Reply:
@allburov 
yes, I know how logging works

can you please mention other devopshq members? If they will agree that additional argument in print function will harm and we have to use logging.debug as default, then I will change the code

I think more opinions is always better :)

Reply:
I'm not sure that it'd be appropriate to ask them to review this minor changes that does not affect project's architecture or other major topic.

I've came across libraries' approaches (boto3 for aws and google api) and they don't provide default progress function at all and just have an example in their docs. 
- https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.download_file
- https://developers.google.com/drive/api/v3/manage-downloads#download_a_document

So I'm asking - why should we? 

I agree that `logging.debug`  can be helpful in debugging, so we can add it as default implementation , but let's keep it simple.

Reply:
that is strange, for example python pip reports download progress, packages on Linux also report progress. 
I thought it will be better for users.

However, if you insist, I change it, it is time to merge this PR :D

Reply:
You're talking about CLI applications, however it's a library. I guess it's a reason why we have a different view.